runtests.sh

```diff
@@ -0,0 +1,2 @@
+#! /bin/bash
+nosetests . -v
```

Comment:
This is a very very simple script I wrote that you can run by calling `./runtests.sh` and it will run all the tests in the code base.

Reply:
When we are ready to do more clever things (like hook in coverage), we'll be able to update this script and it won't change the command you'll have been using to run tests.